hc/front/tests/test_add_channel.py

```diff
@@ -37,5 +37,23 @@ def test_instructions_work(self):
             r = self.client.get(url)
             self.assertContains(r, "Integration Settings", status_code=200)
 
-    ### Test that the team access works
-    ### Test that bad kinds don't work
+    def test_bad_kinds_do_not_work(self):
+        # Test that bad kinds don't work
+        self.client.login(username="alice@example.org", password="password")
+        url = "/integrations/add_not/"
```

Comment:
you could provide a nonsensical kind via a variable such as `bad_kind` = "nonsense"`
and then url = "/integrations/add_%s" % bad_kind